dsc/configurations/reboot-a-node.md

```diff
@@ -0,0 +1,90 @@
+---
+ms.date:  1/17/2019
+keywords:  dsc,powershell,configuration,setup
+title:  Reboot a Node
+---
+# Reboot a Node
+
+> [!NOTE]
+> This topic talks about how to reboot a Node. In order for the reboot to be successful the **ActionAfterReboot** and **RebootNodeIfNeeded** LCM settings need to be configured properly. To read about Local Configuration Manager settings, see [Configure the Local Configuration Manager](../managing-nodes/metaConfig.md), or [Configure the Local Configuration Manager (v4)](../managing-nodes/metaConfig4.md).
```

Comment:
Use semantic line breaks with a max line length of 100. Do this for all paragraphs.

Reply:
Complete